app/views/shared/_gtag.html.haml

```diff
@@ -4,4 +4,7 @@
   window.dataLayer = window.dataLayer || [];
   function gtag(){dataLayer.push(arguments);}
   gtag('js', new Date());
-  gtag('config', "#{ENV['GOOGLE_ANALYTICS']}", { 'anonymize_ip': true, 'linker': { 'accept_incoming': 'true' } });
+  gtag('config', "#{ENV['GOOGLE_ANALYTICS']}", { 'anonymize_ip': true, 'linker': {
+     'domains': ['gov.uk/find-teaching-job', 'gov.uk/guidance/list-a-teaching-job-at-your-school-on-teaching-vacancies'],
```

Comment:
These look like URLs and not domains to me 🤔 .

Reply:
Also the documentation says that you only need to add this config if we're the site which links to those domains (which we aren't).

Reply:
In that case, this is something we should add to `/find-teaching-job` and `/guidance/list-a-teaching-job-at-your-school-on-teaching-vacancies` sites, however, the domain is gov.uk in each of these cases. Feels like this is an all or nothing situation

Reply:
I’m not sure what you’re saying 🤷‍♂️.

Reply:
According to your suggestion, we need to add this code to the sites which link to TVS? But the "domain" itself is `gov.uk` in each case 🤯

Reply:
Yep, if you see Isabelle’s comment in the Trello card there are changes being made to GOV.uk